src/createServer.js

```diff
@@ -1,11 +1,23 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+
+const { router: usersRouter } = require('./routes/users');
+const { clearUser: clearU } = require('./services/usersService');
```

Comment:
better not to minify
```suggestion
const { clearUser } = require('./services/usersService');
```